client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -693,8 +693,8 @@ function getRedirectUrlForPostCheckoutUpsell( {
 		! hasJetpackPlan( cart ) &&
 		! hasDIFMProduct( cart ) &&
 		( hasBloggerPlan( cart ) ||
-			hasPersonalPlan( cart ) ||
-			hasPremiumPlan( cart ) ||
+			hasPersonalPlan( cart ) || // if you are the person who is cleaning up hasPremiumPlan( cart ) case - consider removing this too. My rough assumption - it wasn't even used before
```

Comment:
Unless this is something our team is going to handle soon I'm not sure we should keep that comment - it lacks context (what does _cleaning up_ refers to?) to be really useful.

Reply:
Removed the comment